RELEASE.rst

```diff
@@ -19,9 +19,9 @@ The same branch is used for the release candidated and the final release.
 In the end, the release branch is merged into the main branch.
 
 Update the version to the release candidate with the first being ``rc1`` (as opposed to 0).
-In ``src/towncrier/_version.py`` the version is set using ``incremental`` such as::
+In ``pyproject.toml`` the version is set like::
```

Comment:
I was hoping that we can set the version in single place, like `src/towncrier/_version.py` and the build system can automatically extract the version from there.

Reply:
The changes in `doc` were recycled but now they are no longer relevant. Reverted